packages/commerce-server/src/format-prices-for-product.test.ts

```diff
@@ -65,7 +65,7 @@ for (const quantity of [7, 13, 26, 42]) {
     ])
 
     mockCtx.prisma.merchantCoupon.findFirst.mockResolvedValue(
-      MOCK_SITE_SALE_COUPON,
```

Comment:
this discount is lower than the quantities that are being tested. That's why we need it a higher discount.